core/src/main/scala/org/apache/spark/util/Utils.scala

```diff
@@ -611,9 +613,9 @@ private[spark] object Utils extends Logging {
         }
         Utils.setupSecureURLConnection(uc, securityMgr)
 
-        val timeout = conf.getInt("spark.files.fetchTimeout", 60) * 1000
-        uc.setConnectTimeout(timeout)
-        uc.setReadTimeout(timeout)
+        val timeoutMs = Utils.timeStringAsMs(conf.get("spark.files.fetchTimeout", "60s")).toInt
```

Comment:
Just a question -- do you support "1m"? I can't decide whether that's better than "60s". Kind of reads like "1 megabyte" or something unless it's "1min". Either way seems OK.